### ISSUE TITLE
Using Environment variables for configuration

### ISSUE THREAD

**Issue by mark-sivill-splunk** [commonshost/dohnut#17]
I'm trying to combine the dohnut docker image with another docker image that contains the unbound DNS server using docker-compose.

To configure what the logic can do in a docker container docker-compose uses environment variables.

Currently, dohnut does not accept environment variables directly as part of the configure to get around this either a bash wrapper script could be created for the docker image that passes the environment variables in via the command line, or the node program could be modified to work with environment variables as well as the command line.

**Comment by sebdeckers** [comment]
This could be useful: https://github.com/yargs/yargs/blob/master/docs/api.md#envprefix

**Comment by sebdeckers** [comment]
Nice, it works!

```shell
$ node source/bin.js --listen 127.0.0.1:53535 --doh commonshost
Started listening on 127.0.0.1:53535 (udp4)
Selected https://commons.host
```

```shell
$ DOHNUT_LISTEN=127.0.0.1:53535 DOHNUT_DOH=commonshost node source/bin.js
Started listening on 127.0.0.1:53535 (udp4)
Selected https://commons.host
```

**Comment by sebdeckers** [comment]
Done in https://github.com/commonshost/dohnut/commit/7b675312cbb09d28392829110b633e91f6f69b5c

Released in [v4.6.0](https://github.com/commonshost/dohnut/releases/tag/v4.6.0)
